bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Utils/ElectionGuard.Encryption.Utils.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
following the same pattern as c++ and python, we just break out the generator functions into their own module